config/hd_coldbox_tests/flx_card_hd_coldbox.json

```diff
@@ -0,0 +1,10 @@
+{
+    "flxcardcontroller": {
+        "hardware_map_file": "HardwareMap_VST.txt",
```

Comment:
This looks like the wrong hardware map file name?

Reply:
this is a bug indeed! fixing it now.